project-base/app/tests/FrontendApiBundle/Functional/Product/PaginatedProductsTest.php

```diff
@@ -106,7 +97,7 @@ private function getQueryWithOrderingMode(string $orderingMode): string
         return '
             query {
                 products (
-                    first: 5
+                    first: 4
```

Comment:
That is a problematic topic - the last four include a brand name at the beginning of the name, so I hope it will be sufficient for most, but as I wrote in the description of this PR - I am trying to fix most, but there is no 100% solution.